src/subscriptions/index.ts

```diff
@@ -1,8 +1,10 @@
 const onOuterClick = (node: Node, callback: () => void): (() => void) => {
     const listener = (event: MouseEvent) => {
-        if (!node.contains(event.target as Node)) {
-            callback();
+        const target: Node = event.target as Node;
+        if (node.contains(target) && !node.isEqualNode(target)) {
```

Comment:
Is this deep checking with `isEqualNode` really required ?

Reply:
Yes, because the modal always contains every thing, the only solution which will not affect the others is to check with the equality.